src/plot.js

```diff
@@ -22,6 +22,12 @@ export function plot(options = {}) {
   // Flatten any nested marks.
   const marks = options.marks === undefined ? [] : options.marks.flat(Infinity).map(markify);
 
+  // Prepend grid marks if passed a grid option
+  const gridx = options.x?.grid ?? options.grid;
```

Comment:
We can’t use the nullish coalescing operator here because defaults should only be inherited in the case of undefined, not null.